wmfdata/utils.py

```diff
@@ -9,6 +10,19 @@
 import pandas as pd
 import requests
 
+"""
+Global wmfdata.utils.log logger.  Usage:
+
+from wmfdata.utils import log
+log.info("my message")
+"""
+log = logging.getLogger('wmfdata')
+log.setLevel(logging.INFO)
+formatter = logging.Formatter(fmt='%(asctime)s %(levelname)s %(name)s: %(message)s', datefmt='%Y-%m-%dT%H:%M:%S')
```

Comment:
Is the datetime really necessary? It makes the messages quite long. It's particularly unnecessary in Jupyter since there the messages are clearly tied to the cell that generated them.

Reply:
Using print_err